docs/source/asr/asr_language_modeling.rst

```diff
@@ -457,3 +470,79 @@ You can then pass this file to your flashlight config object during decoding:
            decoding.beam.flashlight_cfg.boost_path='/path/to/my_boost_file.boost' \
            decoding.beam.flashlight_cfg.beam_size_token = 32 \
            decoding.beam.flashlight_cfg.beam_threshold = 25.0
+
+Combine N-gram Language Models
+==============================
+
+Before combining N-gram LMs install required OpenGrm NGram library using `scripts/installers/install_opengrm.sh <https://github.com/NVIDIA/NeMo/blob/stable/scripts/installers/install_opengrm.sh>`__.
+Alternatively, you can build Docker image `scripts/installers/Dockerfile.ngramtools <https://github.com/NVIDIA/NeMo/blob/stable/scripts/installers/Dockerfile.ngramtools>`__ with all the necessary dependencies.
+
+To combine two N-gram language models, you can use the script ngram_merge.py located at 
+`scripts/asr_language_modeling/ngram_lm/ngram_merge.py <https://github.com/NVIDIA/NeMo/blob/stable/scripts/asr_language_modeling/ngram_lm/ngram_merge.py>`__.
+
+This script merges two ARPA N-gram language models and creates a KenLM binary file that can be used with the beam search decoders on top of ASR models.  
+You can specify weights (alpha and beta) for each of the models (ngram_a and ngram_b) correspondingly: alpha * ngram_a + beta * ngram_b.
+The script can also calculate the perplexity of the resulting N-gram language model on a test set.
+Note, each step during the process is cached in the out_path, to speed up further run. To avoid cache use the --force flag.
```

Comment:
out_path -> `out_path` (highlights the word)

Reply:
done